src/api/flaskr/service/profile/profile_manage.py

```
@@ -0,0 +1,639 @@
+def update_profile_item(
+    app: Flask,
+    profile_id: str,
+    key: str,
+    type: int,
+    show_type: int,
+    remark: str,
+    user_id: str,
+    profile_prompt: str = None,
+    profile_check_model: str = None,
+    profile_check_model_args: str = None,
+    items: list[str] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)